src/extensions/react-flow/markers/MarkerArrowClosedInverse.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from "react";
+
+export const MarkerArrowClosedInverse: FC = () => {
```

Comment:
Please add a comment to public/exported components.

Reply:
There will be an new issue to clean a few things up related to this brunch. I will add this request there.